### PR TITLE
Divyanshu | Test | Show Videos in the GIDDH panel in the software

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cu3buz1

### CODE REVIEW

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts [hunk 1]
```diff
@@ -13,23 +13,19 @@ export class WatchVideoComponent implements OnInit {
...
     @Input() public moduleName: string = null;
     /** Holds CSS class for custom code */
     @Input() public cssClass: string = null;
 
     /** Holds final youtube video link  */
     public videoLink: string = '';
+    /* This will hold local JSON data */
+    public localeData: any = {};
+    /** Holds Translated text to display on button */
+    public translatedText = '';
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts]:
```suggestion
    public translatedText: string = '';
```

File: apps/web-giddh/src/app/voucher/voucher.component.html [hunk 2]
```diff
@@ -27,7 +27,9 @@
                             <small *ngIf="invoiceType === 'estimates'"> ({{ commonLocaleData?.app_beta }}) </small>
                         </button>
                     </div>
-                    <watch-video [displayText]="localeData?.watch_video_text" [cssClass]="'text-primary'" [moduleName]="'VOUCHER'"></watch-video>
+                    <ng-container *ngIf="invoiceTypeLabel !== '' && (selectedVoucherType === 'cash' || selectedVoucherType === 'sales')">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/voucher/voucher.component.html]:
```suggestion
                    <ng-container *ngIf="invoiceTypeLabel !== '' && (isCashInvoice || isSalesInvoice)">
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts]: done

File: apps/web-giddh/src/app/voucher/voucher.component.html (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/voucher/voucher.component.html]: done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**